docs/why_mf_next.md

```diff
@@ -85,17 +88,17 @@ But the stability requirements prevent any major cleanup.
 
 ### 3. Hard to map to the existing localization core structures
 
-The format is not well supported by any major localization system. \
+The format is not easy to support in localization tooling.
```

Comment:
No. This is precisely the opposite of what the rest of the paragraph says (and is not correct anyway).

```suggestion
While MessageFormat 1.0 and its syntax are widely supported by runtime environments,
the same cannot be said for localization tooling.